lib/libutils/isoc/bget.c

```diff
@@ -983,6 +985,8 @@ void brel(buf, poolset, wipe)
 	   allocated. */
 
 	register bufsize size = b->bh.bsize;
+        wipe_start = (char *)b;
```

Comment:
I sugget to add a inline comment, like
`/* Only current buffer content, including bfhead, needs to be wiped */`

Reply:
Okey, Comments have been added.